masonite/session.py

```diff
@@ -36,44 +34,39 @@ def set(self, key, value):
 
         ip = self.__get_client_address()
 
-        if not ip in self._session:
+        if not (ip in self._session):
             self._session[ip] = {}
 
         self._session[ip][key] = value
 
-
     def has(self, key):
         """
         Check if a key exists in the session
         """
-        
+
         data = self.__collect_data()
         if data and key in data:
             return True
-        
         return False
 
-
     def all(self):
         """
         Get all session data
         """
 
         return self.__collect_data()
 
-
     def flash(self, key, value):
         """
         Add temporary data to the session
         """
 
         ip = self.__get_client_address()
-        if not ip in self._flash:
+        if not (ip in self._flash):
```

Comment:
Is this really PEP 8? It looks like a useless case of parenthesis. 

(ip in self._flash) is setting a boolean value (True or False) and you are "not"ing it.

maybe we should change it to `if ip not in self._flash` to make it more "Pythonic" (I hate that word)

Reply:
Yes, if ip not in self._flash it's better :)